wolfssl/wolfcrypt/ecc.h

```diff
@@ -91,12 +112,16 @@ typedef struct {
 #endif
 
 #ifndef FP_MAX_BITS_ECC
-    #define FP_MAX_BITS_ECC           528
+    /* This value should be double the max ecc bit size */
+    #define FP_MAX_BITS_ECC       (MAX_ECC_BITS*2)
 #endif
+
 #define FP_MAX_SIZE_ECC           (FP_MAX_BITS_ECC+(8*DIGIT_BIT))
```

Comment:
I think we can remove the 8 fudge digit_bits here.  If FP_MAX_BITS_ECC is not a multiple of digit_bit though we will need to add 1 fudge digit bit.